homeassistant/components/netatmo/netatmo_entity_base.py

```diff
@@ -0,0 +1,124 @@
+"""Base class for Netatmo entities."""
+import logging
+from typing import Dict, List
+
+from homeassistant.core import CALLBACK_TYPE, callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, MANUFACTURER, MODELS
+from .data_handler import NetatmoDataHandler
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class NetatmoBase(Entity):
+    """Netatmo entity base class."""
+
+    DOMAIN = ""
+    TYPE = ""
+
```

Comment:
Maybe I missed it, but these look unused.
```suggestion

```

Reply:
I somehow remember that it cause some trouble when I removed it, but _now_ it doesn't seem to make a difference. And I agree, it makes no sense.